utils/constants.tsx

```diff
@@ -928,6 +929,8 @@ export const CloudLinks = {
     PRORATED_PAYMENT: 'https://mattermost.com/pl/mattermost-cloud-prorate-documentation',
     DEPLOYMENT_OPTIONS: 'https://mattermost.com/deploy/',
     DOWNLOAD_UPDATE: 'https://mattermost.com/deploy/',
+    CLOUD_SIGNUP_PAGE: 'https://mattermost.com/sign-up/',
+    SELF_HOSTED_LOGIN: 'https://customers.mattermost.com/login',
```

Comment:
This should be signup page as well but for self-hosted https://customers.mattermost.com/signup